.gitignore

```diff
@@ -49,4 +49,4 @@ node_modules/
 # Logs
 *.log
 # Desktop Service Store
-*.DS_Store
+*.DS_Store
```

Comment:
Unrelated :)

Reply:
my bad :)

Reply:
but aint that the same thing?

Reply:
i created a python virtual env but then i deleted it and removed it from the gitignore file, everything else is the same. I dont know why its showing that but that shouldnt be a problem .

Reply:
It is about the new line at the end of the file. Your patch removed it.